it/mail/chrome/messenger/prefs.properties

```diff
@@ -1,64 +1,64 @@
 #
 # The following are used by the Account Wizard
 #
-enterAccountName=Please enter a name for this account.
-enterUserName=Please enter a username.
-enterName=Please enter your name.
-enterValidEmail=Inserire un indirizzo e-mail valido.
-accountExists=Un account di posta o dei gruppi di discussione con lo stesso nome utente e server è già presente. Premere Indietro ed inserire un altro server, o premere Annulla.
-modifiedAccountExists=È già presente un account con lo stesso nome utente e nome server. Inserire un differente nome utente e/o nome server.
-userNameChanged=Il proprio nome utente è stato aggiornato. È possibile si voglia aggiornare anche il proprio indirizzo e-mail e/o il nome utente associato con questo account.
-serverNameChanged=Il nome del server è stato modificato. Se si sta utilizzando una configurazione di tipo IMAP verificare che tutte le cartelle utilizzate dai filtri esistano sul nuovo server.
+enterAccountName=Inserisci un nome per questo account.
+enterUserName=Inserisci un nome utente.
+enterName=Inserisci il tuo nome.
+enterValidEmail=Inserisci un indirizzo e-mail valido.
+accountExists=Un account di posta o dei newsgroup con lo stesso nome utente e server è già presente. Premi Indietro e inserisci un altro server, o premi Annulla.
+modifiedAccountExists=È già presente un account con lo stesso nome utente e nome server. Inserisci un differente nome utente e/o nome server.
```

Comment:
CORRECTION NEEDED. Please replace with the following: 

+modifiedAccountExists=È già presente un account con lo stesso nome utente e nome server. Inserisci un nome utente e/o nome server diversi.